tools/package_gcsfuse/package.go

```diff
@@ -37,7 +37,7 @@ func packageFpm(
 		"-v", version,
 		"-d", "fuse",
 		"--vendor", "",
-		"--maintainer", "Aaron Jacobs <jacobsa@google.com>",
+		"--maintainer", "GCSFuse dev <gcs-fuse-dev@google.com>",
```

Comment:
**For reviewer:** I couldn't find any usage of "package_gcsfuse". It would be helpful to mention its usage here if you know of any and if there exist any. We can add the same in the comments/internal docs.

Reply:
I am not aware either. Did you check the readme/description deb/rmp packages or on cloud rapture.

Reply:
From reading the description, I understand that this is used for creating gcsfuse package. However, in our current release, we don't use code under package_gcsfuse. Rather we use code under package_gcsfuse_docker. Anyways, I will merge these changes.